### PR TITLE
SD-1480 Create Teletype Text Format

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: server/superdesk/io/teletype.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                logger.exception("Ingest Type: Teletype - File: {0} could not be processed".format(filename))
+                self.move_file(self.path, filename, provider=provider, success=False)
+                raise ParserError.ZCZCParserError(ex, provider)
+            except ParserError as ex:
+                self.move_file(self.path, filename, provider=provider, success=False)
+            except Exception as ex:
+                self.move_file(self.path, filename, provider=provider, success=False)
+                raise ProviderError.ingestError(ex, provider)
+
+        push_notification('ingest:update')
```
Comment (petrjasek) [inline server/superdesk/io/teletype.py]: no need for this, it will be called after `_update`

**Comment by petrjasek** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by actionless** [comment]
figure out why tests were failing

probably u need to pull from master first

**Comment by amagdas** [comment]
@marwoodandrew Please reopen this PR.

**Pull request merged**